web/app/views/index.scala.html

```diff
@@ -662,8 +662,7 @@
             {{/dataset-schema}}
           </div>
           <div id="ownertab" class="tab-pane">
-            {{#dataset-author owners=owners ownerTypes=ownerTypes showMsg=showMsg alertType=alertType ownerMessage=ownerMessage parentController=this}}
-            {{/dataset-author}}
+            {{dataset-author owners=owners ownerTypes=ownerTypes showMsg=showMsg alertType=alertType ownerMessage=ownerMessage parentController=this}}
```

Comment:
Suggestion, to improve readability you could have each attribute on its own line:

{{dataset-author 
    owners=owners 
    ownerTypes=ownerTypes 
    showMsg=showMsg 
    alertType=alertType 
    ownerMessage=ownerMessage 
    parentController=this}}